packages/glimmer-demos/lib/ripples.ts

```diff
@@ -79,18 +79,14 @@ export function start() {
     data.points.push( new Point(e.timeStamp, e.offsetX, e.offsetY) );
   });
 
-  let baseOffset = Date.now() - window.performance.now();
```

Comment:
I'm not sure what the intention here was (it wasn't working), please let me know if we do need an offset and I'll get it working